File: Assets/PACKT_Scripts/FPSAnimation.js
#pragma strict
var thisAnimator : Animator;

function Start () 
{
	thisAnimator = GetComponent(Animator);
}

function Update()
{
	if(Input.GetButton("Fire1"))
	{
		thisAnimator.SetTrigger("Shooting");
		Shoot();
	}
}

function Shoot()
{

}

function Pick()
{
	thisAnimator.SetTrigger("Picking");
}

File: Assets/PACKT_Scripts/FPSAnimation.cs
using UnityEngine;
using System.Collections;

public class FPSAnimation : MonoBehaviour {
	public Animator theAnimator;

	void Start () 
	{
		theAnimator = GetComponent<Animator>();
	}
	
	void Update () 
	{
		if(Input.GetButton ("Fire1"))
		{
			theAnimator.SetTrigger ("Shooting");
			Shoot();
		}
	}
	void Shoot ()
	{

	}
}
